tldr/diagnostics.py

```
@@ -963,6 +941,34 @@ def get_project_diagnostics(
     if not path.exists():
         return {"error": f"Path not found: {project_path}", "diagnostics": []}
 
+    # Languages with project-level diagnostic dispatch below
+    _DIAG_LANGUAGES = {"python", "typescript", "go", "rust", "ruby", "elixir"}
+
+    # Handle "all": detect project languages, run diagnostics per language, merge
+    if language == "all":
+        detected = _detect_project_languages(path)
+        languages = [l for l in detected if l in _DIAG_LANGUAGES] or ["python"]
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)